andela/lotus-ah-frontend
# #160451558 implement login functionality

#### What does this PR do?
 - Implementing login functionality

#### Description of Task to be completed?
- a user should be able to log in

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app should open on port ```8080``` on your browser
navigate to the login page

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160451558](https://www.pivotaltracker.com/story/show/160451558, "Users should be able to Login to Authors Haven")

#### Screenshots (if appropriate)
<img width="528" alt="screen shot 2018-11-06 at 10 57 38 pm" src="https://user-images.githubusercontent.com/20081126/48097215-82201300-e219-11e8-8453-bbd599bf921a.png">



#### Questions:
